addons/im_livechat/models/discuss_channel.py

```diff
@@ -66,6 +66,7 @@ class DiscussChannel(models.Model):
     )
     chatbot_current_step_id = fields.Many2one('chatbot.script.step', string='Chatbot Current Step')
     chatbot_message_ids = fields.One2many('chatbot.message', 'discuss_channel_id', string='Chatbot Messages')
+    chatbot_current_step_is_last = fields.Boolean(compute="_compute_chatbot_current_step_is_last")
```

Comment:
this field is a bit redundant (also it could be just a related on `chatbot_current_step_id.is_last`, but we can probably live without it)